src/com/view/bar/bar.js

```diff
@@ -145,6 +146,45 @@ export default class ViewBar extends Component {
 		}
 	}
 
+	/**
+	 * Adds the # of notifications in the favicon.
+	 * Shows individual numbers until 9, then 9+ for more.
+	 */
+	refreshFavIconNotificationCount() {
+		const notificationCount = this.state.notifications || 0;
+		// remove existing icon if any
+		const el = document.querySelector("link[rel='shortcut icon']");
+		if (el) el.parentNode.removeChild(el);
+		// reload LD favicon and paint it
+		const canvas = document.createElement('canvas');
+		canvas.setAttribute('width', 64);
+		canvas.setAttribute('height', 64);
+		const context = canvas.getContext('2d');
+		const icon = new Image();
+		icon.setAttribute('src', '/favicon.ico');
+		icon.onload = () => {
+			// LD favicon as background
+			context.drawImage(icon, 0, 0);
+			// We only draw text for # > 0
+			if (notificationCount > 0) {
+				const text = notificationCount > 9 ? "9+" : String(notificationCount);
+				// Dark backdrop for contrast
+				context.fillStyle = '#0d4263';
```

Comment:
The colors should probably be from the color-theme (when running the dev environment you can find them here http://ludumdare.org/dev/palette)

Reply:
Great idea! Fixed in 71d9966